python-sdk/src/astro/sql/operators/transform.py

```diff
@@ -128,14 +157,13 @@ def transform_file(
         or will overwrite a table given in the `output_table` parameter.
     """
 
-    @transform(
+    return TransformOperator(
         conn_id=conn_id,
         parameters=parameters,
+        handler=None,
         database=database,
         schema=schema,
+        sql=file_path,
+        python_callable=lambda: (file_path, parameters),
```

Comment:
So would this not run into the same scaling issues that the decorated tasks run into? Have you tested this at scale?

Reply:
I am 100% sure that it won't run into the scaling issues, because I tested `transform` decorator vs `TransformOperator`.

Reply:
Out of curiosity, what are the numbers for both approaches?

Reply:
See https://github.com/astronomer/astro-sdk/pull/897#discussion_r986817532